src/main/kotlin/org/rust/cargo/runconfig/CargoTestRunState.kt

```diff
@@ -29,7 +29,8 @@ class CargoTestRunState(
 
     private val cargoTestPatch: CargoPatch = { commandLine ->
         val rustcVer = cargoProject?.rustcInfo?.version
-        commandLine.copy(additionalArguments = patchArgs(commandLine, rustcVer))
+        // TODO: always pass `withSudo` when `com.intellij.execution.process.ElevationService` supports error stream redirection
+        commandLine.copy(additionalArguments = patchArgs(commandLine, rustcVer), withSudo = false)
```

Comment:
Let's notify users that sudo in tests is not supported
something like
```kotlin
if (commandLine.withSudo) {
    environment.project.showBalloon("Sudo is not supported in tests yet", NotificationType.WARNING)
}
```
but with more user-friendly text

Reply:
Done, see https://github.com/intellij-rust/intellij-rust/pull/7113#issuecomment-856655170